connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskTest.java

```diff
@@ -161,6 +165,7 @@ public class WorkerSinkTaskTest {
     private StatusBackingStore statusBackingStore;
     @Mock
     private KafkaConsumer<byte[], byte[]> consumer;
+    private Consumer<byte[], byte[]> mockConsumer;
```

Comment:
Nit: it's a little strange to call this `mockConsumer` when there's also a `consumer` field that itself contains a mocked consumer. To avoid confusion, can we just make this a local variable for any of the tests that require it?

Reply:
Yeah, I have made this a local variable inside that test method.